_episodes/09-conflict.md

```diff
@@ -47,21 +47,22 @@ $ cat mars.txt
 Cold and dry, but everything is my favorite color
 The two moons may be a problem for Wolfman
 But the Mummy will appreciate the lack of humidity
-This line added to Wolfman's copy
+An underground base with few windows would be best, to minimise Wolfman's exposure to the moons
+A windbreak could protect a base from dust storms
 ~~~
 {: .output}
 
 and then push the change to GitHub:
 
 ~~~
 $ git add mars.txt
-$ git commit -m "Add a line in our home copy"
+$ git commit -m "Add discussion of base type and weather protections"
```

Comment:
I'd prefer something that's in the story but is easy for a novice learner to see that this line was added from Wolfman's copy.